manifests/overlays/single-namespace/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+# This overlay patches in KubeRay operator configuration
+# necessary for KubeRay single namespace support.
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../../base
+patches:
+- path: patch-cr.json
```

Comment:
What about the name? Will we have `cluster-role` created in namespace scope?

Reply:
this will convert all cluster-role to role with namespace.